core/src/replay_stage.rs

```diff
@@ -2382,10 +2383,7 @@ impl ReplayStage {
 
         info!(
             "{} reset PoH to tick {} (within slot {}). {}",
-            my_pubkey,
-            bank.tick_height(),
-            bank.slot(),
-            next_leader_msg,
+            my_pubkey, tick_height, slot, next_leader_msg,
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#uninlined_format_args

Reply:
Good point - these are no longer mixed. Gonna BP so cleaned it up real fast